src/common/websocket/config.js

```diff
@@ -19,8 +20,8 @@ const domain_config = {
         app_id: 16929,
     },
     local: {
-        hostname: 'localhost',
-        app_id: 17044,
+        hostname: 'localhost.deriv.com',
```

Comment:
@cakasuma Could you explain again why we need `deriv.com` instead of `localhost` I don't really understand it either yet 😅

Reply:
same question :)

Reply:
I meant `localhost`

Reply:
I see I thought why its deriv.com 😂 . there was no concrete reason why its localhost.deriv.com, I was discussing with @negarn and I think we were inspired by deriv app and bot as it was `local`.binary.sx. Hence, localhost only should be fine as well. it is up to us developer to decide the name actually since this is only local preview.